composer.json

```diff
@@ -11,7 +11,7 @@
         "spryker/kernel": "^3.56.0",
         "spryker/log": "^3.7.0",
         "spryker/queue": "^1.5.0",
-        "spryker/store": "^1.10.0",
+        "spryker/store": "^1.21.0",
```

Comment:
why do we need this and Testify update in the first place?

Reply:
because this is the fix for Trait "SprykerTest\Shared\Store\Helper\StoreDataHelperTrait" not found behavior for prefer lowest pipeline in the master branch